Main.java

```diff
@@ -1,3 +1,4 @@
+import java.util.Scanner;
 /**
  *
  * @author 

```

Comment:
Don't forget to put your name and the program description inside of the code header!